python/tskit/drawing.py

```diff
@@ -221,7 +236,284 @@ def remap_style(original_map, new_key, none_value):
         return str(text_tree)
 
 
-class SvgTreeSequence:
+class Plotbox:
```

Comment:
CI currently failing on Python 3.6 because dataclass is only available from Python >= 3.7. I guess by the time this PR is reviewed and merged we'll have [changed the min reqs to Python 3.7](https://github.com/tskit-dev/tskit/issues/1171) anyway.

Reply:
Just merged! https://github.com/tskit-dev/tskit/pull/1235

Reply:
Fast work, thanks @benjeffery !